packages/core/src/tracing/transaction.ts

```diff
@@ -288,7 +289,9 @@ export class Transaction extends SpanClass implements TransactionInterface {
       },
       spans: finishedSpans,
       start_timestamp: this.startTimestamp,
+      // eslint-disable-next-line deprecation/deprecation
       tags: this.tags,
+      extra: spanGetAttributes(this),
```

Comment:
Currently, data from the transaction was not actually ever sent 😬 I guess it is OK to send this as extra?